src/components/BorrowerProfile/LoanDescription.vue

```diff
@@ -1,118 +1,149 @@
 <template>
 	<section>
-		<h2
-			class="tw-text-h2"
-		>
-			{{ firstName }}'s story
-		</h2>
-		<div
-			v-if="partnerName"
-		>
-			<div>
-				{{ moreInfoAboutLoan }}
-			</div>
-			<div
-				v-if="loanAlertText"
+		<div>
+			<h2
+				class="tw-text-h2"
+				v-if="isAnonymizationLevelFull"
 			>
-				<h3
-					class="tw-text-h3"
-				>
-					About {{ partnerName }}:
-				</h3>
+				Story
+			</h2>
+			<h2
+				class="tw-text-h2"
+				v-if="!isAnonymizationLevelFull"
+			>
+				{{ borrowerOrGroupName }}'s story
+			</h2>
+		</div>
+		<div>
+			<section v-if="storyDescription">
 				<p
-					v-for="(paragraph, index) in loanAlertTextParagraphs"
+					v-for="(paragraph, index) in storyDescriptionParagraphs"
 					:key="index"
 					class="tw-text-base"
 				>
 					{{ paragraph }}
 				</p>
-			</div>
-			<div
-				v-if="dualStatementNote"
-			>
-				<h3
-					class="tw-text-h3"
-				>
-					Important Note About This Loan
-				</h3>
-				<p>
-					{{ dualStatementNote }}
+			</section>
+
+			<section v-if="borrowersList">
+				<p class="tw-text-base">
+					{{ borrowersList }}
 				</p>
-			</div>
-		</div>
-		<div
-			v-if="!partnerName"
-		>
-			<h3
-				class="tw-text-h3"
-			>
-				Business Description
-			</h3>
-			<p
-				v-for="(paragraph, index) in businessDescriptionParagraphs"
-				:key="index"
-				class="tw-text-base"
-			>
-				{{ paragraph }}
-			</p>
-			<h3
-				class="tw-text-h3"
-			>
-				What is the purpose of this loan?
-			</h3>
-			<p
-				v-for="(paragraph, index) in purposeParagraphs"
-				:key="index"
-				class="tw-text-base"
-			>
-				{{ paragraph }}
-			</p>
+			</section>
+
+			<section>
+				<!--
+				"Previous Loan sections" omitted. Needs separate ticket
+				-->
+			</section>
+
+			<section v-if="storyTranslation">
+				<img v-if="reviewerImageLink" src="{{ reviewerImageLink }}" alt="Translator profile picture">
```

Comment:
Suggest adding a lazy loading attribute
`<img loading="lazy" ...`
since this image is further down the page